index.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="./src/styles.css">
+    <title>Document</title>
+    <h1>About Me</h1>
+</head>
+<body>
+        <P>Interested in merging poetry and photography</P>
+       <img src=https://images.unsplash.com/photo-1474204075013-fafcfee9bfd7?ixlib=rb-1.2.1&auto=format&fit=crop&w=2125&q=80>
+    <h2>My favorite activities:</h2>
+        <ul>
+        <li>Night swimming</li>
+        <li>Stargazing</li>
+        <li>Running in the woods</li>
+        </ul>
+    <h3>Books I like:</h3>
```

Comment:
Since this is a similar header to the h2 tag I would just use another h2 here. If you had a nested header inside your list then that's when it would be best to use the h3.